tools/mvn.bzl

```diff
@@ -0,0 +1,287 @@
+# Copyright (C) 2018 The Dagger Authors.
```

Comment:
This and other file in this PR could be easily made generic,
for example, `dagger_pom_file` could be named `maven_pom_file`,
and extracted to https://github.com/bazelbuild/bazel-skylib instead
of putting it inside Dagger.

Reply:
The goal is definitely for these to be put in a common location. IIUC bazel-skylib is meant for lower-level Skylark functions, but I'm open to putting this wherever makes most sense.

The vision I had was for `pom_file` to be the base implementation that's exported and `dagger_pom_file` is a project-specific convenience function that specifies some defaults. Every project could decide to use this, or just use `pom_file` directly.